nuxeo-java-client/src/main/java/org/nuxeo/client/api/objects/Document.java

```diff
@@ -278,6 +289,9 @@ public String getName() {
     }
 
     public void set(String key, Object value) {
+        if (value instanceof Calendar || value instanceof Date) {
```

Comment:
I don't find it consistent to retrieve something with `#get` I didn't set with `#set`. Either we handle `Date` and `Calendar` in input and output, either we just handle `String`.
```java
Calendar cal = new Calendar();
doc.set('dc:issued', cal);
cal = doc.get('dc:issued') // won't work
```